meta-ostro/conf/distro/ostro.conf

```diff
@@ -206,6 +205,7 @@ PNWHITELIST_meta-java += " \
 PNWHITELIST_meta-soletta += " \
     soletta \
     soletta-dev-app \
+    libmicrohttpd \
```

Comment:
@mythi, I have tested this patch to see if Soletta will build with http-server support.

Everything worked however, I had to use the patch solettaproject/meta-soletta#48 or yours here.

Thank you very much